stock_picking_line_sequence/views/stock_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+        <record id="view_picking_form" model="ir.ui.view">
+            <field name="name">stock.picking.form</field>
+            <field name="model">stock.picking</field>
+            <field name="inherit_id" ref="stock.view_picking_form" />
+            <field name="priority" eval="255" />
+            <field name="arch" type="xml">
+                <xpath
+                expr="//notebook/page/field[@name='move_ids_without_package']"
+                position="before"
+            >
+                    <field name="max_line_sequence" invisible="1" />
+                </xpath>
+                <xpath
+                expr="//notebook/page/field[@name='move_ids_without_package']/tree/field[@name='name']"
+                position="before"
+            >
+                    <field name="sequence" widget="handle" />
+                    <field name="sequence2" />
+                </xpath>
+                <xpath
+                expr="//notebook/page/field[@name='move_ids_without_package']"
+                position="attributes"
+            >
+                    <attribute
+                    name="context"
+                >{'picking_type_code': picking_type_code, 'default_picking_id': id,
+                    'form_view_ref':'stock.view_move_form', 'address_in_id': partner_id,
+                    'default_picking_type_id': picking_type_id, 'default_location_id': location_id,
+                    'default_location_dest_id': location_dest_id, 'default_sequence': max_line_sequence}
+                    </attribute>
```

Comment:
Overwriting the context in the form will conflict (functionally) with other modules that need to insert a default value in the same place. Would you consider something like this to inject the additional context key on the code level?

```
    @api.model
    def get_view(self, view_id=None, view_type='form', **options):
        """Append the default sequence.

        The context of `move_ids_without_package` is already overloaded and replacing
        it in a view does not scale across other extension modules.
        """
        res = super().get_view(view_id=view_id, view_type=view_type, **options)
        if res.get('arch') and view_type == 'form':
            doc = etree.XML(res['arch'])
            elements = doc.xpath("//field[@name='move_ids_without_package']")
            if elements:
                element = elements[0]
                context = element.get('context', '{}')
                context = '%s%s, %s' % (context[0], "'default_sequence': max_line_sequence", context[1:])
                element.set('context', context)
            res['arch'] = etree.tostring(doc, encoding='unicode')
        return res
```